### ISSUE TITLE
Missing Libs

### ISSUE THREAD

**Issue by BearzRobotics** [Zowja/New-Frontier-Craft#3]
So I have NFC code up in eclipse and it says its missing minecraft.jar, jinput.jar, lwjgl_util.jar and lwjgl.jar. Presumably they are in my minecraft directory some where, but having a wiki page that explains were to pull and then put these files.

**Comment by Zowja** [comment]
Ah those are apart of normally setting up MCP, so you'll want to put the bin file created from playing Beta from the old launcher into /jars. Then decompile. If you've already done that and decompiled, then add /jars/bin/[all jars here] to your referenced libraries

**Comment by BearzRobotics** [comment]
Yeah that fixed it, I went into the json file next to b1.7.3.jar and found the path to each of the jar files. Since this only has the code that you modified how would I go about running the game. If I just bulk copy the other files in great that gets it going, but how would one easily pull out all of the files that were modified leaving the rest to recommit.
